### DISCUSSION TITLE
import css is fail

### DISCUSSION THREAD

**Discussion by bluelovers** [vercel/next.js#19110] [Help]
# Bug report

## Describe the bug

![image](https://user-images.githubusercontent.com/167966/98972704-bee01c80-254d-11eb-8bdd-3859766c3fd0.png)

---

https://nextjs.org/blog/next-10#importing-css-from-third-party-react-components

![image](https://user-images.githubusercontent.com/167966/98973085-3f9f1880-254e-11eb-9bfb-21c462de4009.png)

## To Reproduce

```tsx
import 'react-big-calendar/lib/sass/styles.scss';
```

## System information

"react": "^17.0.1",

- OS: [e.g. macOS, Windows]
- Browser (if applies) [e.g. chrome, safari]
- Version of Next.js: [e.g. 6.0.2] 10.0.2-canary.12
- Version of Node.js: [e.g. 10.10.0] v15.1.0

**Comment by bluelovers** [comment]
https://codesandbox.io/s/nice-mendeleev-oeuw7?file=/pages/index.js

**Comment by lordvcs** [comment]
I have the same issue while trying to use 'swiper' npm package. Using `import 'swiper/swiper.scss';` gives me the same error message 

> error - ./node_modules/swiper/swiper.scss
Global CSS cannot be imported from files other than your Custom <App>. Please move all global CSS imports to pages/_app.page.tsx. Or convert the import to Component-Level CSS (CSS Modules).

**Reply by RyanPaiva56** [reply]
Did you find a solution for this? I'm having the same issue with swiper.

**Reply by lordvcs** [reply]
No, I ended up importing to the global css file as `@import '~swiper/swiper.scss'`
